packages/nextjs/utils/sanityImage.ts

```diff
@@ -0,0 +1,9 @@
+import { q, sanityImage } from "groqd";
+
+export const sanityImagesSelection = (): any =>
+  process.env.NEXT_PUBLIC_API_MOCKING === "enabled"
+    ? q("images").filter().deref()
+    : sanityImage("images", { isList: true, withCrop: true, withHotspot: true });
```

Comment:
@gksander This is unfortunate, but when using mock data, even though the JSON is fully formed, `sanityImage` doesn't `deref` correctly. So we need to manually force a deref here for mock mode only. This does break type safety as we have to "any" this. 

Considering the amount of maintenance required around the mock e2e tests, we should lean towards removing them and leaning on the real API-data ones.